src/go/rpk/pkg/cli/cluster/config/get.go

```diff
@@ -40,10 +40,20 @@ output, use the 'edit' and 'export' commands respectively.`,
 			client, err := adminapi.NewClient(fs, p)
 			out.MaybeDie(err, "unable to initialize admin client: %v", err)
 
-			currentConfig, err := client.Config(cmd.Context(), true)
+			currentConfig, err := client.SingleKeyConfig(cmd.Context(), key)
 			out.MaybeDie(err, "unable to query current config: %v", err)
 
 			val, exists := currentConfig[key]
+			if !exists {
+				//If len(currentConfig) != 0, it contains an alias to the property we specified.
+				//Attempt to iterate over the map, assign the value, and set exists to true.
+				for _, v := range currentConfig {
+					val = v;
+					exists = true;
+					break
+				}
+			}
+
```

Comment:
[question] IIUC, if you query for a key `foo` but it has an alias `bar` it will return:

```
# Query for foo
{"bar":"the value"}
```

Is there a case where a SingleKeyConfig will return a map with a length that is different than 0 or 1?

Reply:
If key `foo` is an alias for property `bar`, yes, the returned JSON is `{"bar":"the value"}`.

If property `bar` is queried, then the JSON is also `{"bar":"the value"}`.

> Is there a case where a SingleKeyConfig will return a map with a length that is different than 0 or 1?

No. This wouldn't be possible due to the filter applied in `server.cc`: https://github.com/redpanda-data/redpanda/pull/22674/files#diff-654649200c71f39ef00f42627dbb15c5280a6f5481301d4cc3ef8a847985e3e6R1280-R1287

But, I can add a sanity check just in case.

Reply:
Updated the behavior here.

New behavior is perhaps closer to what is expected:

If key `foo` is an alias for property `bar`, the returned JSON is `{"foo":"bar_value"}`.

If property `bar` is queried, then the JSON is also `{"bar":"bar_value"}`